# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/MediaDecoderStateMachine.cpp
# Commit: 569b8d3a9fcd
# Full Hash: 569b8d3a9fcd96eb276c5225f81fcf9c01183565
# Author: Paul Adenot <paul@paul.cx>
# Date: 2022-05-10 03:50:31
# Regressor Bug: 1743834
# File Overlap Count: 1
# Description:
#   Bug 1743834 - Split initialization and start in AudioSinkWrapper. r=alwu
#   
#   No change in functionality, this is to prepare for the next patch that will do
#   the initialization asynchronously because it's expensing and blocking.
#   
# ==============================================================================

diff -r 9c37cd8faac3 -r 569b8d3a9fcd dom/media/MediaDecoderStateMachine.cpp
--- a/dom/media/MediaDecoderStateMachine.cpp	Mon May 09 13:20:52 2022 +0000
+++ b/dom/media/MediaDecoderStateMachine.cpp	Mon May 09 13:20:53 2022 +0000
@@ -2884,10 +2884,10 @@
   }
 
   RefPtr<MediaDecoderStateMachine> self = this;
-  auto audioSinkCreator = [self](const media::TimeUnit& aStartTime) {
+  auto audioSinkCreator = [self]() {
     MOZ_ASSERT(self->OnTaskQueue());
     AudioSink* audioSink =
-        new AudioSink(self->mTaskQueue, self->mAudioQueue, aStartTime,
+        new AudioSink(self->mTaskQueue, self->mAudioQueue,
                       self->Info().mAudio, self->mSinkDevice.Ref());
     self->mAudibleListener.DisconnectIfExists();
     self->mAudibleListener = audioSink->AudibleEvent().Connect(